src/main/kotlin/uk/gov/justice/digital/hmpps/nomisuserrolesapi/config/NomisUserRolesApiExceptionHandler.kt

```diff
@@ -157,14 +161,28 @@ class NomisUserRolesApiExceptionHandler {
       )
   }
 
+  @ExceptionHandler(ForbiddenRoleAssignmentException::class)
+  fun handleForbiddenRoleAssignmentException(e: ForbiddenRoleAssignmentException): ResponseEntity<ErrorResponse?>? {
+    log.debug("Forbidden Role assignment: {}", e.message)
+    return ResponseEntity
+      .status(FORBIDDEN)
+      .body(
+        ErrorResponse(
+          status = FORBIDDEN,
+          userMessage = "Role assignment forbidden: ${e.message}",
```

Comment:
Just wondering if we're potentially exposing too much info here?

Reply:
it should never get to the endpoint - the UI wont display the role to add is the admin doesn't have the role - it was just belt and braces approach